benchmark/benchmark/local.py

```diff
@@ -52,7 +52,8 @@ def run(self, debug=False):
             sleep(0.5)  # Removing the store may take time.
 
             # Recompile the latest code.
-            cmd = CommandMaker.compile().split()
+            cmd = CommandMaker.compile(mem_profiling = self.mem_profile)
+            Print.info(f"About to run {cmd}...")
```

Comment:
Please reflect in `benchmark` if you mean to keep this.

Reply:
Not sure what you mean by that.... mem_profile is a new instance variable based on the config, so this change should be fine....

Reply:
I'm sorry, I misspoke: I meant please reflect, in `remote`, I.e. here: https://github.com/MystenLabs/narwhal/blob/3d921e757acdadf3e61fbf32b151e23f08d89ad2/benchmark/benchmark/remote.py#L169-L170

You're documenting the memory options very well in the `README`, but this `README` documents in a single explanation that you can run the benchmarks in both  `local` and `remote` modes, with the assumption those run more or less identically.

It would be great to therefore have the ability to run remote memory profiling ... or document explicitly that this will only work in `local` mode